lib/src/repository/traits/thermostat_repository.dart

```diff
@@ -30,9 +30,17 @@ class ThermostatRepository {
         FanMode(trait.state.fanMode.reported?.value ?? AvailableFanMode.ON),
         ThermostatMode(
             trait.state.mode.reported?.value ?? AvailableThermostatMode.OFF),
+        AmbientTemperature(
+            trait.state.ambientTemperature.reported?.value ?? 0.0),
```

Comment:
I had similar comments in @rlperez's pr: I don't think this should default to 0.0.  Seems like it should be null.  I'm imagining things like temperature freeze warnings misbehaving if we are defaulting to a numerical value. @yonomi-french any thoughts?

Reply:
If the team decides on 'Null' I'm good with that. From my standpoint, it sounds like the least ambiguous and prone to error.

Reply:
I have changes on my PR to make these nullable

Reply:
Please reference the Slack thread on this topic. Let's proceed with the "Target Temperature Unavailable" text for the front-end.